app/views/govuk_publishing_components/components/_attachment.html.erb

```diff
@@ -56,10 +56,23 @@
             data: data_attributes %>
     <% end %>
 
+    <% if attachment.reference.present? %>
+      <%= tag.p "Ref: #{attachment.reference}", class: "gem-c-attachment__metadata gem-c-attachment__metadata--compact" %>
+    <% end %>
+
+    <% if attachment.unnumbered_reference.present? %>
+      <%= tag.p attachment.unnumbered_reference, class: "gem-c-attachment__metadata gem-c-attachment__metadata--compact" %>
+    <% end %>
+
     <% if attributes.any? %>
       <%= tag.p raw(attributes.join(', ')), class: "gem-c-attachment__metadata" %>
     <% end %>
 
+    <% if attachment.is_official_document %>
+      <%= tag.p link_to("Order a copy", "https://www.gov.uk/guidance/how-to-buy-printed-copies-of-official-documents", class: "govuk-link", target: "_blank"),
```

Comment:
Sorry to be late - should we set this as a link with no visited state since it's a bit confusing that if you visit one of these ever it looks like you've ordered a copy of every attachment.

Reply:
Addressed by https://github.com/alphagov/govuk_publishing_components/pull/1427